.gitignore

```diff
@@ -3,3 +3,4 @@ composer.lock
 phpunit.xml
 tests/Fixtures/project/var/cache/*
 build/
+.idea
```

Comment:
Please, don't add IDE-specific folders into the project's .gitignore. Configure global gitignore instead, see https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore

Reply:
fixed